ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/data/CustomerCenterState.kt

```diff
@@ -5,16 +5,39 @@ import com.revenuecat.purchases.PurchasesError
 import com.revenuecat.purchases.customercenter.CustomerCenterConfigData
 import com.revenuecat.purchases.ui.revenuecatui.customercenter.dialogs.RestorePurchasesState
 
-internal sealed class CustomerCenterState {
-    object Loading : CustomerCenterState()
-    data class Error(val error: PurchasesError) : CustomerCenterState()
+internal sealed class CustomerCenterState(
+    open val dismissCustomerCenter: Boolean = false,
```

Comment:
not sure if there's a better way of doing this, but I think this is the most correct.

Reply:
[`573399b` (#2010)](https://github.com/RevenueCat/purchases-android/pull/2010/commits/573399ba232e88d0830cf5c863660ace1d12169e) I think this looks much better